src/js/shared/application-form.js

```diff
@@ -133,6 +133,13 @@ exports.createApplicationForm = function createApplicationForm(validateFile = tr
       ],
       cssClasses,
     }),
+    student_status: fields.boolean({
+      label: 'Please confirm your student status:',
+      note: '',
+      widget: checkboxWidget('I am currently a student, or by the event I would have graduated after January 28, 2016.'),
```

Comment:
This wording is a little confusing. Couldn't it just be "I am currently a student or I graduated after January 28th 2016"? i.e. If they are graduating between now and the event, they are currently a student right?